library/rsa.c

```diff
@@ -1268,6 +1268,93 @@ int mbedtls_rsa_public(mbedtls_rsa_context *ctx,
     return 0;
 }
 
+#if !defined(MBEDTLS_RSA_NO_CRT)
```

Comment:
We're fixing a performance regression in an LTS branch, so this needs a changelog entry.